client/sites-dashboard/utils.ts

```diff
@@ -1,7 +1,7 @@
 import { SiteExcerptNetworkData } from 'calypso/data/sites/site-excerpt-types';
 
 export const getLaunchpadUrl = ( slug: string, flow: string ) => {
-	return `/setup/launchpad?flow=${ flow }&siteSlug=${ slug }`;
+	return `/setup/${ flow }/launchpad?siteSlug=${ slug }`;
```

Comment:
A couple other options to consider:
1. Create some common `getLaunchPadUrl()` utility and use it throughout the codebase, so we aren't maintaining our own version.
2. Link to `/home/${ slug }` and let the Launchpad logic determine where to redirect them.

Thoughts?

Reply:
I'd go with 1 since it's more explicit and less magical around how `/home/${ slug }` works. Also, the behavior in 2 might change in the future, or My Home might disappear. :)

Looping in @alshakero for thoughts on the utility.

Reply:
@vindl Sounds good. I created an issue for it: https://github.com/Automattic/wp-calypso/issues/70909

@alshakero Can you delegate the issue?